docker/grafana/setup-grafana

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+
+
+def psql_sql(sql):
+    p = subprocess.Popen(["psql", "-h", "postgres", "-U", "chroma", "-tAc {}".format(sql)], stdout=subprocess.PIPE)
+    return p.stdout.read()
+
+
+out = psql_sql(
```

Comment:
Where was this done in the RPM based workflow?

Reply:
https://github.com/whamcloud/integrated-manager-for-lustre/blob/master/chroma_core/lib/service_config.py#L590-L592